build.gradle

```diff
@@ -122,24 +131,40 @@ bintray {
 }
 
 ext {
-    springCloudVersion = 'Finchley.M9'
+    githubFeignVersion = '10.1.0'
```

Comment:
do we need to be on 9.6 on clients?
i.e. this is:
https://github.com/hmcts/service-auth-provider-java-client/blob/master/build.gradle#L136

Perhaps we start with updating this one but I have vague feelings we had problems when we tried to put one client on 10.x

Reply:
after running through most of the commits i have realised perhaps better approach would have been to bump to spring boot 2.0.x and feign v9, tag it and do next batch of spring boot 2.1.x and feign v10. shall i review commits like that instead of jumping straight to latest?

yep, didn't see problems. `@deprecated` methods have been removed but they are not used anywhere in the project